src/components/Inputs/input.module.scss

```diff
@@ -1469,13 +1465,6 @@
 
         .expandable-thumb {
             position: absolute;
-            left: -4px;
```

Comment:
I hope you've tested this. Earlier, the position was required because of the expandable transition effect.

Reply:
Yes, tested and it's good with the smaller size.